tests/unit/test_vcf_to_gnomad.py

```diff
@@ -0,0 +1,34 @@
+
+
+def generate_gnomad_id(vcf_line):
+    # Assuming a standard VCF format with tab-separated fields
+    fields = vcf_line.strip().split('\t')
+
+    # Extract relevant information (you may need to adjust these indices based on your VCF format)
+    chromosome = fields[0]
+    position = fields[1]
+    reference_allele = fields[3]
+    alternate_allele = fields[4]
+
+    # Create a gnomAD-like ID
+    gnomad_id = f"{chromosome}-{position}-{reference_allele}-{alternate_allele}"
+
+    return gnomad_id
+
+
+def test_generate_gnomad_id(my_translator):
```

Comment:
I see you're using the test vcf input that works for 6/8 variants, is this test supposed to fail? I'm getting:

> tests/unit/test_vcf_to_gnomad.py [('chr19-54220024-G-*,A', ValueError('Unable to parse data as gnomad variation')), ('chr19-54220999-A-T', ValidationError('Expected reference sequence A on GRCh38:chr19 at positions (54220998, 54220999) but found T')), ('chr19-54221654-A-T,P', ValidationError('Expected reference sequence A on GRCh38:chr19 at positions (54221653, 54221654) but found T'))]

Reply:
It fails the same way as the original test `test_vcf_annotation`. no?